src/Avalonia.Controls/Platform/Win32CornerHints.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+
+namespace Avalonia.Platform;
+
+/// <summary>
+/// Specifies hints for window corner appearance.
+/// </summary>
+[Flags]
+public enum Win32WindowCornerHints
```

Comment:
Why is this flags? And plural naming?

Reading the docs I don't think multiple values can be set at once. So it's isn't treated as flags.

https://learn.microsoft.com/en-us/windows/win32/api/dwmapi/ne-dwmapi-dwmwindowattribute
https://learn.microsoft.com/en-us/windows/win32/api/dwmapi/ne-dwmapi-dwm_window_corner_preference

Perhaps the name should be `Win32WindowCornerPreference` as well. Although, I think it's supported only on Windows 11+ so calling it "Hints" might be appropriate as its ignored on platforms that don't support it.